admin/RunIncrementalJSONDump

```diff
@@ -43,6 +47,7 @@ echo Disabling all database triggers
 echo Making incremental JSON dumps
 ./admin/DumpIncrementalJSON \
     --output-dir "$TEMP_DIR" \
+    --tmp-dir "$(make_temp_dir --extra "$DUMP_DIR")" \
```

Comment:
Would this directory be cleaned up/removed?  I don't think I see where it would be.  Just want to make sure we're not accidentally syncing temporary files to FTP.

Reply:
With `--extra` it doesn't set up the rmdir trap AFAICT, though I'm not sure that'd work anyway if there are still empty subdirectories.

Reply:
OK, I forgot about the rsync `--exclude`, so syncing them shouldn't be an issue, but maybe they aren't deleted still.